docs: Prometheus Cluster Metrics (#1239)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md

8. Confirm your changes by selecting **Confirm & Create**.

9. Click on **Add New Pack**.

10. Use the following information to find the Spectro Cluster Metrics pack.
    - **Pack Type**: Monitoring
    - **Registry**: Public Repo
    - **Pack Name**: Spectro Cluster Metrics
    - **Pack Version**: 3.3.X or newer.

11. Use the default values. Confirm your changes by selecting **Confirm & Create**.

12. Click on **Next** to review the cluster profile. Save the cluster profile.

13. Navigate to the left **Main Menu** and select **Clusters**.

14. Click on **Add New Cluster**. Select **Deploy New Cluster**.

15. Pick the infrastructure provider you selected for the cluster profile you created earlier. 

> 16. Go ahead and assign a name to the host cluster and select the registered account you wish to deploy the host cluster to.  Click on **Next**.

Reviewer: ```suggestion
16. Assign a name to the host cluster and select the registered account that will deploy it.  Click on **Next**.
```

Reviewer: Or "Assign a name to the host cluster and select the registered account to deploy it."

> 17. Pick a cluster profile for your cluster. Click on **Next**.

Reviewer: ```suggestion
17. Select a cluster profile to apply to your cluster. Click on **Next**.
```

> 18. The next screen displays all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above all the cluster profile layers.

Reviewer: ```suggestion
18. The next screen displays all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above the cluster profile layers.
```

8. Confirm your changes by selecting **Confirm & Create**.

9. Click on **Add New Pack**.

> 10. Use the following information to find the Prometheus Operator pack.

Reviewer 2: 10. "... to find the Spectro Cluster Metrics pack"

and 

6. "... to find the Prometheus Agent pack"

8. Confirm your changes.

9. Click on **Add New Pack**.

10. Use the following information to find the Spectro Cluster Metrics pack.
    - **Pack Type**: Monitoring
    - **Registry**: Public Repo
    - **Pack Name**: Spectro Cluster Metrics

>     - **Pack Version**: 3.3.X or newer.

Reviewer: ```suggestion
    - **Pack Version**: 3.3.X or newer
```

11. Use the default values. Confirm your changes by selecting **Confirm & Create**.

12. Click on **Next** to review the cluster profile. Save the cluster profile.

13. Navigate to the left **Main Menu** and select **Clusters**.

14. Click on **Add New Cluster**. Select **Deploy New Cluster**.

15. Pick the infrastructure provider you selected for the cluster profile you created earlier. 

> 16. Go ahead and assign a name to the host cluster and select the registered account you wish to deploy the host cluster to.  Click on **Next**.

Reviewer: ```suggestion
16. Assign a name to the host cluster and select the registered account that will deploy it.  Click on **Next**.
```

Reviewer: Or "Assign a name to the host cluster and select the registered account to deploy it."

> 17. Pick a cluster profile for your cluster. Click on **Next**.

Reviewer: ```suggestion
17. Select a cluster profile to apply to your cluster. Click on **Next**.
```

> 18. The following screens will display all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above all the cluster profile layers.

Reviewer: ```suggestion
18. The next screen displays all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above the cluster profile layers.
```
